lens/lotus/api.go

```diff
@@ -23,3 +30,87 @@ type APIWrapper struct {
 func (aw *APIWrapper) Store() adt.Store {
 	return aw.store
 }
+
```

Comment:
A couple thoughts on the change: this is definitely an improvement, but I have some concerns --- some of which you already mentioned:
- Are we going to grow this as the api consumed by Visor grows? Or is there some alternative?
- Since we are wrapping the API would it be better to have it as a field instead of embedding?
- Would it be more appropriate to add API tracing to lotus instead? (as a follow on later)
- Should we include the arguments to these calls as tags on the Span?

Reply:
We'll need to grow this if we want complete tracing coverage (we may not) or we should attempt to get it into lotus instead. I'm not convinced we want every method covered, especially since we are concerned about performance.

Wrapping the api, making it unexported would help enforce some type safety.

We could include the arguments as tags. I don't know useful it would be in general so I avoided paying the marshalling cost of things like cids in this pass.